3.process_cfret_features/extract_sc_features.ipynb

```diff
@@ -61,18 +61,26 @@
     "# Set paths for plate localhost220512140003_KK22-05-198\n",
```

Comment:
<p>I would update this documentation.</p><p> </p><p>My suggestion is:</p><p> </p><p>There are three plates from the CFReT data:</p><p> </p><ol><li>localhost220512140003_KK22-05-198 - contains images from well columns 1-8 (only wells that did not have phalloidin blow out)</li><li>localhost220513100001_KK22-05-198_FactinAdjusted - contains images from well columns 9-12 (wells that were adjusted for the phalloidin blow out)</li><li>localhost230405150001</li></ol><p> </p><p>The first two plates images together make a full 384-well plate, but they are kept in their separate plates in case there are any batch effects. Plate 3 is a different plate with different conditions (see README for more details).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/CFReT_data/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>